packages/blockchain-link/src/workers/blockbook/websocket.ts

```diff
@@ -196,8 +197,19 @@ export class BlockbookAPI extends EventEmitter {
         // create deferred promise
         const dfd = createDeferred<void>(-1);
 
-        // initialize connection
-        const ws = new WebSocket(url);
+        // TODO: create new message to set TOR proxy
+        const agent =
+            url.indexOf('.onion') > 0 ? SocksProxyAgent('socks://127.0.0.1:9050') : undefined;
```

Comment:
But this will route only .onion addresses through Tor, right? If I set custom blockbook instance with common url in Suite, it will bypass Tor proxy even when Tor is on, won't it?

Reply:
solved by 1f85612